plugins/modules/aci_interface_policy_fc.py

```diff
@@ -212,6 +244,11 @@ def main():
         fc_policy=dict(type="str", aliases=["name"]),  # Not required for querying all objects
         description=dict(type="str", aliases=["descr"]),
         port_mode=dict(type="str", choices=["f", "np"]),  # No default provided on purpose
+        auto_max_speed=dict(type="str", choices=["2G", "4G", "8G", "16G", "32G"]),
```

Comment:
should we create a constant for for these speeds? and then add as list from the constant. Since this is partially re-used in speed?